### DISCUSSION TITLE
Follow Me!

### DISCUSSION THREAD

**Discussion by DevXcode1** [esin/esin#14] [Guestbook]
Hey,
Follow me on Github :)
Love from Los Angeles!

**Comment by esin** [comment]
Hey

Done

**Comment by alex-semenyuk** [comment]
Hey @esin, 
Great Github account!
It would be nice to see you in my followers as well.
Thanks!

**Comment by DevilishDawn431** [comment]
@esin i need help starting the scratch vr project, so will you join the team?

**Reply by gooddavvy** [reply]
I know I am not @esin, but I'd be happy to help! Please tell me how to join.

**Comment by gooddavvy** [comment]
I would love to get followed, too, @esin. Do you already follow me?

**Reply by esin** [reply]
Yep :)

**Reply by gooddavvy** [reply]
@esin Alright, thank you! :D I actually follow a lot of people just like you do. But why did you decide to do that? Following many people is like following the entire community, getting a bunch of notifications!

**Reply by esin** [reply]
First of all - Because I can :)

And the second - why not? Github gives well documented API, but didn't write how many followings I can have :D Just want to test

**Comment by gooddavvy** [comment]
> First of all - Because I can :)
> 
> And the second - why not? Github gives well documented API, but didn't write how many followings I can have :D Just want to test

@esin Okay :)

**Comment by gooddavvy** [comment]
@esin Oh and don't forget to follow @0xxtiger

**Comment by ivan-developer-01** [comment]
@esin Can you follow me please

**Reply by ivan-developer-01** [reply]
@esin Thanks! Sorry for being so unkind in first time.

**Comment by Kevin-Prichard** [comment]
Greetings!
